app/models/pro_account.rb

```diff
@@ -13,4 +13,8 @@
 class ProAccount < ActiveRecord::Base
   belongs_to :user,
              :inverse_of => :pro_account
+
+  def stripe_customer
+    Stripe::Customer.retrieve(stripe_customer_id)
```

Comment:
What does a failure look like? Is there always a `stripe_customer_id`? (in theory there probably should be) Can we save an API lookup if there isn't?